app/pages/sections/supplier/search/template.test.js

```diff
@@ -23,6 +23,31 @@ describe('supplier search page', () => {
     });
   }));
 
+  it('should render error summary with correct error text and hrefs if there are errors', componentTester(setup, (harness) => {
+    const context = {
+      questions: [
+        {
+          id: 'supplierName',
+          error: [{ message: 'some supplier name error message' }],
+        },
+      ],
+      errors: [
+        { text: 'some supplier name error message', href: '#supplierName' },
+      ],
+    };
+
+    harness.request(context, ($) => {
+      const errorSummary = $('[data-test-id="error-summary"]');
+      const errorArray = $('[data-test-id="error-summary"] li a');
+      expect(errorSummary.length).toEqual(1);
+      expect(errorArray.length).toEqual(context.errors.length);
+      context.errors.forEach((error, i) => {
+        expect(errorArray[i].attribs.href).toEqual(error.href);
+        expect(errorArray[i].children[0].data.trim()).toEqual(error.text);
+      });
+    });
+  }));
+
```

Comment:
do we need tests for error messages around the text input field?

Reply:
Cool I'll add something around this 👍